code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -238,6 +238,9 @@
 	var/ignore_pain = FALSE // when true the rav will not go into crit or take crit damage.
 	var/ignore_pain_state = 0 // how far "dead" the rav has got while ignoring pain.
 
+	// *** Tindalos vars ***
+	var/obj/effect/alien/warp_beacon/warp_beacon = null
```

Comment:
needs some autodoc explaining what this is.

Reply:
Done.